runners/flink/src/main/java/org/apache/beam/runners/flink/LyftFlinkStreamingPortableTranslations.java

```diff
@@ -118,17 +122,45 @@ private void translateKafkaInput(
     }
 
     final String topic;
-    Preconditions.checkNotNull(topic = (String) params.get("topic"), "'topic' needs to be set");
+    final String groupId;
+    final String bootstrapServers;
 
+    Preconditions.checkNotNull(topic = (String) params.get("topic"), "'topic' needs to be set");
     Map<?, ?> consumerProps = (Map) params.get("properties");
     Preconditions.checkNotNull(consumerProps, "'properties' need to be set");
-    final Properties properties = new Properties();
-    properties.putAll(consumerProps);
 
-    logger.info("Kafka consumer for topic {} with properties {}", topic, properties);
+    logger.info("Configuring kafka consumer for topic {} with properties {}", topic, consumerProps);
+
+    Preconditions.checkNotNull(
+        groupId = (String) consumerProps.remove("group.id"), "'group.id' needs to be set");
+    Preconditions.checkNotNull(
+        bootstrapServers = (String) consumerProps.remove("bootstrap.servers"),
+        "'bootstrap.servers' needs to be set");
+
+    final String splits[] = bootstrapServers.split(":", 2);
+    final String bootstrapServer = splits[0];
+    int port = splits.length == 2 ? Integer.parseInt(splits[1]) : DEFAULT_KAFKA_BROKER_PORT;
```

Comment:
This should probably be done in the builder.

Reply:
I thought about it.  I strongly feel the builder should just take `bootstrapServers` as an argument which could be a list of `bootstrapServer:port, [bootstrapServer:port]....`  . 

Though https://github.com/lyft/streamingplatform/blob/master/spflinkjava/streamingplatform-kafka/src/main/java/com/lyft/streamingplatform/LyftKafkaProducerBuilder.java#L90 is a public API, given no services have. adopted the builders, it's best time to break compatibility now and fix it.  Will update the builder code.

Reply:
We should also support multiple servers. It looks like we just support one at the moment.

Reply:
+1 for both of the changes. This should be as simple as possible for user code.

Reply:
Done! Pushed a lot of heavy lifting code here to the builder.